project.clj

```diff
@@ -1,7 +1,15 @@
-(defproject circleci/bond "0.4.0"
+(defproject circleci/bond "0.5.0"
   :description "Spying library for testing"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies []
   :plugins [[lein-cloverage "1.2.2"]]
-  :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.2"]]}})
+  :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.2"]]}}
+  :repositories [["releases" {:url "https://clojars.org/repo"
+                              :username :env/clojars_username
+                              :password :env/clojars_token
+                              :sign-releases false}]
```

Comment:
Curious what's needed for us to use signed releases.

Reply:
If it's anything like how Maven does it then we'd need to publish a PGP key tied to CircleCI (e.g. `oss@circleci.com`) and then set up the necessary keychain in CI. The most painful part in my experience is finding a good process for managing and rotating the PGP key.

Reply:
and IIRC signatures aren't validated by Maven by default so the valuing in signing these would just be to folks who choose to enable validation